pages/about.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CC Global Summit 2015</title>
+    <link rel="stylesheet" href="../styles/style.css" />
+    <link rel="stylesheet" href="../styles/about/style.css" />
+    <link rel="icon" href="../images/icons/favicon.ico" type="image/x-icon" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="desktop-navbar">
+        <div class="navbar-container">
+          <ul>
+            <li>
+              <a href="#"><img src="../images/navbar/facebook.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="../images/navbar/twitter.png" alt="" /></a>
+            </li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+          </ul>
+        </div>
+        <div class="navbar">
+          <div class="navbar-logo">
+            <a href="../index.html">
+              <img src="../images/navbar/logo.png" alt="logo" />
+            </a>
+          </div>
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li>CC Campaing</li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="mobile-navbar">
+        <div class="close-button-contaienr">
+          <img
+            src="../images/icons/close_mobile_menu_btn.png"
+            alt="close-button"
+          />
+        </div>
+        <div class="nav-items">
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="hero-section">
+        <div class="menu-icon-container">
+          <img
+            src="../images/icons/mobile_menu_button.png"
+            alt="menu-icon"
+            id="menu-icon"
+          />
+        </div>
+        <div class="hero-title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
```

Comment:
- [ ] @belalgamal46 great work implementing the changes requested, as previously highlighted by the previous reviewer kindly ensure your header elements follow a hierarchy for better SEO and semantics of the page. For more info check this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes)